src/components/Disputes/DisputeVoided.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+function DisputeVoided({ id }) {
+  return (
+    <div>
+      <h2>Void Notice for Dispute #{id}</h2>
```

Comment:
here i think that we need to check if the voided text is an url and if it is we should display a link to the blogspot or whatever cc @sohkai

Reply:
Actually we are going to have a text and a link for each voided dispute in the `voided-disputes` file. 
And we will render the data like this https://www.figma.com/file/IdVA6iMn0cWLV5imylwEUP/%E2%9C%85-MASTER-Court-Dashboard?node-id=3930%3A344